Content.Server/GameObjects/Components/Explosion/ClusterFlashComponent.cs

```diff
@@ -0,0 +1,180 @@
+#nullable enable
+using Content.Shared.Interfaces.GameObjects.Components;
+using Content.Server.GameObjects.Components.Explosion;
+using Robust.Shared.GameObjects;
+using System.Threading.Tasks;
+using Robust.Server.GameObjects.Components.Container;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Serialization;
+using System;
+using System.Diagnostics.CodeAnalysis;
+using Content.Server.GameObjects.Components.Trigger.TimerTrigger;
+using Content.Server.Throw;
+using Robust.Server.GameObjects;
+using Content.Shared.GameObjects.Components.Explosion;
+using Content.Shared.Physics;
+using Robust.Shared.GameObjects.Components;
+using Robust.Shared.GameObjects.Components.Timers;
+using Robust.Shared.Interfaces.Random;
+using Robust.Shared.IoC;
+using Robust.Shared.Map;
+using Robust.Shared.Maths;
+using Robust.Shared.Random;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Server.GameObjects.Components.Explosives
+{
+    [RegisterComponent]
+    public sealed class ClusterFlashComponent : Component, IInteractUsing, IUse
+    {
+        public override string Name => "ClusterFlash";
+
+        private Container _grenadesContainer = default!;
+
+        /// <summary>
+        ///     What we fill our prototype with if we want to pre-spawn with grenades.
+        /// </summary>
+        [ViewVariables]
+        private string? _fillPrototype;
+
+        /// <summary>
+        ///     Maximum grenades in the container.
+        /// </summary>
+        [ViewVariables]
+        private int _maxGrenades;
+
+        [ViewVariables]
+        private int _maxViewableGrenades;
```

Comment:
This is unnecessary given the visualizer doesn't even use it.

Reply:
Component and visualizer use this variables.

ClusterFlash
line 63 _maxGrenades use 
line 172 _maxViewableGrenades use (ClusterFlashVisuals.GrenadesMax)

Reply:
_maxViewableGrenades become useless. Deleted.